server/storage/backend/tx_buffer.go

```diff
@@ -52,7 +54,20 @@ func (txw *txWriteBuffer) put(bucket Bucket, k, v []byte) {
 }
 
 func (txw *txWriteBuffer) putSeq(bucket Bucket, k, v []byte) {
-	// TODO: Add (in tests?) verification whether k>b[len(b)]
+	// putSeq is only be called for the data in the Key bucket. The keys
```

Comment:
Could we merge verify independent of the solution?

Reply:
Raised PR https://github.com/etcd-io/etcd/pull/17325